src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -87,7 +73,21 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
    * Indicates whether to disable the country field.
    */
   @Input()
-  public disabled: boolean = false;
+  public set disabled(isDisabled: boolean) {
```

Comment:
I noticed toggling the disabled state will reset the values for pristine and touched. For example, load the form then edit one of the inputs so it is no longer pristine. Then, click the button to disable the forms and you'll notice pristine gets reset back to `true` and touched back to `false`. I tried this with a basic reactive input and did not have the same results. Is it supposed to do this?

Reply:
👀

Reply:
I just added a normal text input inside the form and see the same behavior where if I edit that input and then disable that input the outer form reverts to pristine?

Reply:
Happy to hop on a call with you if it is easier but I'm seeing the same behavior with the basic input as with the country field.

Reply:
Ok, I double checked this and it turns out I had a nested form group that was causing issues. You're right - this is the expected Angular behavior. Thanks for checking!